bot/cogs/eval.py

```diff
@@ -31,7 +29,8 @@ def __init__(self, bot: Bot):
 
         self.interpreter = Interpreter(bot)
 
-    def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
+    def _format(self, inp: str, out: Any) -> Tuple[str, Optional[discord.embed]]:
```

Comment:
Just noticed a type when running this file.

```suggestion
    def _format(self, inp: str, out: Any) -> Tuple[str, Optional[discord.Embed]]:
```